course/models/models.py

```diff
@@ -162,6 +162,11 @@ def has_view_permission(self, user):
     def has_edit_permission(self, user):
         return user.is_teacher
 
+    def get_duplicate_question(self):
```

Comment:
You can rename this function to `clone(self)`.
Then you can use question.clone() to duplicate a question.